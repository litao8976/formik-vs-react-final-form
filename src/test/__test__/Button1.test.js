import renderer from 'react-test-renderer'
import React from 'react';
import Button1 from '../Button1';

describe('Button1', () => {
    it('matches snapshot', () => {
        const component = renderer.create((
            <Button1 />
        ))
        expect(component.toJSON()).toMatchSnapshot()
    })
})
    