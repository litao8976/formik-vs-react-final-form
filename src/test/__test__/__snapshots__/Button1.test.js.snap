// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button1 matches snapshot 1`] = `
<div>
  <button>
    xsdadas
  </button>
</div>
`;
